app/views/users/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>A given User detail with posts</h1>
\ No newline at end of file
+<h1>A given User detail with posts</h1>
+<%= render 'shared/user', user:@user%>
+
+<div class="user_bio">
+ <h3>Bio</h3>
+ <p><%= @user.bio %><p>
+</div>
+
+<div class="user_post">
+   <% @user.most_three_recent_post.each do |post| %>
```

Comment:
- [ ] Kindly fix the typo for the user model method, this method name should be the same as the one in the user model for the app to work.

![method-error](https://github.com/alicemirigo92/blog_post/assets/53124548/30ad42e7-a5e5-4db2-b517-6ec3b2f5141a)

Reply:
- [ ] Well resolved 🥇 
https://github.com/alicemirigo92/blog_post/blob/4d144e60418f6c59294f9a70796a4f2b543aea1d/app/views/users/show.html.erb#L10